packages/palette-docs/src/components/GlobalComponents.jsx

```diff
@@ -101,28 +101,40 @@ export const MarkdownComponents = {
   },
   h1: props => (
     <Box mb={5}>
-      <Serif size="8" color="black100">
+      <Serif element="h1" size="8" color="black100" className="DocSearch-lvl1">
```

Comment:
This seems retroactively like a good rationale for @dblandin's [suggestion](https://artsy.slack.com/archives/C9XJKPY9W/p1569448740010100)